Lib9c.Policy/Policy/BlockPolicySource.cs

```diff
@@ -287,7 +290,7 @@ public partial class BlockPolicySource
                     $"{((ITransaction)transaction).CustomActions?.Count}",
                     transaction.Id);
             }
-            else if (IsObsolete(transaction, actionTypeLoader, index))
+            else if (IsObsolete(transaction, actionTypeLoader, improperIndex))
```

Comment:
How about using `properIndex + 1` (or just `index + 1`) and leaving a comment here? (I think `improperIndex` term can be a little burden to understand)